src/engine/GroupByHashMapOptimization.h

```diff
@@ -42,11 +42,13 @@ struct AvgAggregationData {
   // _____________________________________________________________________________
   [[nodiscard]] ValueId calculateResult(
       [[maybe_unused]] const LocalVocab* localVocab) const {
-    if (error_)
-      return ValueId::makeUndefined();
-    else
-      return ValueId::makeFromDouble(sum_ / static_cast<double>(count_));
+    if (error_) return ValueId::makeUndefined();
+    if (count_ == 0) return ValueId::makeFromInt(0);
```

Comment:
Please use {braces} after the `if` , I forgot this in the review of the student who wrote that.
And write a short comment that it is according to the standard (`AVG(empty) == 0`)